.github/workflows/ci.yml

```diff
@@ -63,6 +63,18 @@ jobs:
         run: yarn install --immutable
       - name: 💅 Prettier Check
         run: yarn prettier:check
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Your code is great. Can we also add icons to their names?
```
name: 🏗️ Build app
```

This is one of the CI files I like to reference: https://github.com/CruGlobal/give-web/blob/bf5a90d45a79961e57e788ad869afe765ed568d5/.github/workflows/build.yml

_(I think I sent you to MPDX, when I meant to say Give)_